website/content/docs/enterprise/admin-partitions.mdx

```diff
@@ -308,7 +309,7 @@ Verify that your Consul deployment meets the [Kubernetes Requirements](#kubernet
 1. Install the workload client clusters:
 
   ```shell-session
-  $ helm install ${HELM_RELEASE_CLIENT} hashicorp/consul --version "0.43.0" --create-namespace --namespace consul --values client.yaml
+  $ helm install ${HELM_RELEASE_CLIENT} hashicorp/consul --version "0.48.0" --create-namespace --namespace consul --values client.yaml
```

Comment:
Should this be 0.49.x or 0.50.0?

Reply:
We can bump it to 0.49.0 since that will be released very shortly